labs/a9.md

```diff
@@ -206,34 +208,46 @@ including signatures and certificates for digital identity verification.
 Suppose you are an important public entity (maybe your pseudonym is Natoshi
```

Comment:
```In [lab a4](a4#generating-and-using-ssh-keys), one task was to generate an SSH
key, using the command:
```
This link is broken

Reply:
I just tried this link and it seems to work for me